perfkitbenchmarker/benchmark_spec.py

```diff
@@ -191,11 +191,24 @@ def Prepare(self):
   def Delete(self):
     if FLAGS.run_stage not in ['all', 'cleanup'] or self.deleted:
       return
+
     if self.vms:
-      vm_util.RunThreaded(self.DeleteVm, self.vms)
-    self.firewall.DisallowAllPorts()
+      try:
+        vm_util.RunThreaded(self.DeleteVm, self.vms)
+      except Exception as e:
+        logging.error('Got an exception: %s\n'
```

Comment:
I agree with this.  More info is better!

On Tue, Apr 28, 2015 at 7:41 AM, Connor McCoy <notifications@github.com>
wrote:

> In perfkitbenchmarker/benchmark_spec.py
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/224#discussion_r29249115>
> :
>
> >      if self.vms:
> > -      vm_util.RunThreaded(self.DeleteVm, self.vms)
> > -    self.firewall.DisallowAllPorts()
> > +      try:
> > +        vm_util.RunThreaded(self.DeleteVm, self.vms)
> > +      except Exception as e:
> > +        logging.error('Got an exception: %s\n'
>
> Consider logging.exception for these as well so the stack trace is
> printed, unless you don't think it will be informative.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/224/files#r29249115>
> .
>



-- 
Anthony F. Voellm (aka Tony)
Google Voice:  (650) 516-7382
<https://www.google.com/voice/b/0?pli=1#phones>
Blog: http://perfguy.blogspot.com/